rechunker/api.py

```diff
@@ -1,19 +1,16 @@
 """User-facing functions."""
 import html
 import textwrap
-import uuid
 
 import zarr
 import dask
-import dask.array as dsa
-from dask.optimization import fuse
-from dask.delayed import Delayed
-
 
 from rechunker.algorithm import rechunking_plan
+import rechunker.dask
+from rechunker.types import CopySpec, StagedCopySpec
 
 
-class Rechunked(Delayed):
+class Rechunked:
```

Comment:
Can you remove the "Methods" line from the docstring? I just needed that so that sphinx / numpydoc didn't complain.

Reply:
done!